vmc/resource_vmc_sddc.go

```diff
@@ -184,6 +184,32 @@ func resourceSddc() *schema.Resource {
 				ValidateFunc: validation.IntBetween(3, 16),
 				Description:  "The maximum number of hosts that the cluster can scale out to.",
 			},
+			"microsoft_licensing_config": {
+				Type: schema.TypeList,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"mssql_licensing": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The status of MSSQL licensing for this SDDC’s clusters. Possible values : ENABLED or DISABLED.",
+							ValidateFunc: validation.StringInSlice([]string{
+								LicenseConfigEnabled, LicenseConfigDisabled,
+							}, false),
+						},
+						"windows_licensing": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The status of Windows licensing for this SDDC's clusters. Possible values : ENABLED, DISABLED or CUSTOMER'S ",
```

Comment:
For the description on these fields also I have reached out to the VMC team for providing more detailed description on what is meant by "CUSTOMER's" for the windows licensing field

Reply:
@rchadalawada I still have not received an answer from VMC team. Waiting for their confirmation